plugins/woocommerce-admin/client/shipping/shipping-recommendations.tsx

```diff
@@ -103,7 +103,11 @@ const ShippingRecommendations: React.FC = () => {
 		select( PLUGINS_STORE_NAME ).getActivePlugins()
 	);
 
-	if ( activePlugins.includes( 'woocommerce-services' ) ) {
+	if (
+		activePlugins.includes( 'woocommerce-services' ) ||
+		activePlugins.includes( 'woocommerce-shipping' ) ||
+		activePlugins.includes( 'woocommerce-tax' )
+	) {
```

Comment:
Practically the same things [as here](https://github.com/woocommerce/woocommerce/pull/48701/files#r1652681640) but we have to do this here too because this place in the code governs a different variant of the banner. Which variant you see depends on the `shipping-smart-defaults` feature flag.

<img width="720" alt="Screenshot 2024-06-06 at 13 42 54" src="https://github.com/woocommerce/woocommerce/assets/1759681/3af1c0dc-c194-4c43-8989-fd7cfc648194">